bouncer.go

```diff
@@ -22,20 +23,29 @@ import (
 )
 
 const (
-	crowdsecLapiHeader      = "X-Api-Key"
-	crowdsecCapiHeader      = "Authorization"
-	crowdsecLapiRoute       = "v1/decisions"
-	crowdsecLapiStreamRoute = "v1/decisions/stream"
-	crowdsecCapiLogin       = "v2/watchers/login"
-	crowdsecCapiStreamRoute = "v2/decisions/stream"
-	cacheTimeoutKey         = "updated"
+	crowdsecLapiHeader         = "X-Api-Key"
+	crowdsecCapiHeader         = "Authorization"
+	crowdsecLapiDecisionsRoute = "v1/decisions"
+	crowdsecLapiStreamRoute    = "v1/decisions/stream"
+	crowdsecLapiLoginRoute     = "v1/watchers/login"
+	crowdsecCapiLoginRoute     = "v2/watchers/login"
+	crowdsecCapiStreamRoute    = "v2/decisions/stream"
+	cacheTimeoutKey            = "updated"
+	captchaSiteVerifyURL       = "https://www.google.com/recaptcha/api/siteverify"
+	decisionTypeBan            = "ban"
+	decisionTypeCaptcha        = "captcha"
+	decisionTypeThrottle       = "throttle"
 )
 
 //nolint:gochecknoglobals
 var (
 	isStartup               = true
 	isCrowdsecStreamHealthy = true
 	ticker                  chan bool
+	captchaSecretKey        = ""
+	captchaSiteKey          = ""
+	captchaVerifyRoute      = ""
+	jwtLapiToken            = ""
```

Comment:
You declare variables globaly, it's fine but you use bouncer's one.
Choose one way of doing this.
Prefer the bouncer one `//nolint:gochecknoglobals` GO don't really like global vars

Reply:
good point, thanks!